test/unit/BondingManager.js

```diff
@@ -2372,6 +2372,27 @@ contract("BondingManager", accounts => {
         })
     })
 
+    describe("setCurrentRoundTotalActiveStake", () => {
+        const transcoder = accounts[0]
+
+        beforeEach(async () => {
+            await fixture.roundsManager.setMockBool(functionSig("currentRoundInitialized()"), true)
+            await fixture.roundsManager.setMockBool(functionSig("currentRoundLocked()"), false)
```

Comment:
Don't think we need this line for this test.

Reply:
fixed in e6d92cf